petastorm/tests/test_pytorch_dataloader.py

```diff
@@ -127,3 +126,26 @@ def test_decimal_friendly_collate_input_has_decimals_in_tuple():
     assert len(actual) == 2
     assert desired[0] == actual[0]
     np.testing.assert_equal(desired[1], actual[1].numpy())
+
+
+@pytest.mark.parametrize('reader_factory', MINIMAL_READER_FLAVOR_FACTORIES)
+def test_no_shuffling(synthetic_dataset, reader_factory):
+    with DataLoader(reader_factory(synthetic_dataset.url, schema_fields=['^id$'], workers_count=1,
+                                   shuffle_row_groups=False)) as loader:
+        ids = [row['id'][0].numpy() for row in loader]
+        np.testing.assert_array_equal(range(len(ids)), ids)
```

Comment:
I don't really understand this check. Should we be check the len(ids) vs the len(synthetic_dataset)?

Reply:
You are right. Made it clearer.

Reply:
Okay, the tests are clearer now.